installation/synology.md

```diff
@@ -3,10 +3,113 @@ layout: documentation
 title: Synology DiskStation
 ---
 
-# Synology DiskStation
+# Synology DiskStation DSM 7.0 and higher
 
-The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages.
-We are proud to be able to provide an [openHAB Synology package](https://github.com/openhab/openhab-syno-spk).
+The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages. For versions of DSM before 7.0 please refer to the second half of this page.
+
+## Prerequisite - prepare the Synology OS for openhab
+
+Synology NAS is based on unix but it has limitations since it is not a full server. There are some shortcuts you have to take to make it work. For the use of openHAB you need some preparation steps. For this create a ssh session into your NAS. For reference how to do that: I found this [youtube tutorial](https://www.youtube.com/watch?v=BCCIMRbAUp8).
```

Comment:
My Synlogy NAS has a Linux kernel.

```suggestion
Synology NAS is based on Linux but it has limitations since it is not a full server.
There are some shortcuts you have to take to make it work.
For the use of openHAB you need some preparation steps.
For this create a ssh session into your NAS.
For reference how to do that: I found this [youtube tutorial](https://www.youtube.com/watch?v=BCCIMRbAUp8).
```